src/transformers/models/wav2vec2/modeling_wav2vec2.py

```diff
@@ -115,19 +120,21 @@ class Wav2Vec2ForPreTrainingOutput(ModelOutput):
     codevector_perplexity: torch.FloatTensor = None
     hidden_states: Optional[Tuple[torch.FloatTensor]] = None
     attentions: Optional[Tuple[torch.FloatTensor]] = None
+    contrastive_loss: Optional[torch.FloatTensor] = None
+    diversity_loss: Optional[torch.FloatTensor] = None
 
 
 def _compute_mask_indices(
```

Comment:
This function is changed in a breaking way, but it was always private and never used outside of the model for non-pretraining tasks, so that should be fine. 

Still have to verify that all the ASR downstream fine-tuning still gives same good results? Do we have to check `audio-classification` as well - does it make use of `mask_time_prob`? @anton-l